OpenRA.Game/Orders/UnitOrderGenerator.cs

```diff
@@ -17,6 +17,9 @@ namespace OpenRA.Orders
 {
 	class UnitOrderGenerator : IOrderGenerator
 	{
+		private const string SelectCursor = "select";
```

Comment:
This also ought not be `const` if you want it to be configurable in the way I stated earlier in a top-level comment.